packages/query-core/src/__tests__/queryClient.test.tsx

```diff
@@ -1050,7 +1050,7 @@ describe('queryClient', () => {
         queryKey: key1,
         queryFn: () => 'data',
       })
-      queryClient.fetchQuery({
+      queryClient.prefetchQuery({
```

Comment:
this now throws a `CancelledError` because we call `cancelQueries` with `revert:false`, and since we don’t revert, the error makes it to the cache. It did so before too, but since the refactor to async/await, it shows up as unhandled in node.